hardware/icm20948-breakout/NOTICE

```diff
@@ -0,0 +1,3 @@
+#ATTRIBUTION
+Thanks to [dtornqvist](https://github.com/dtornqvist/icm-20948-breakout) for a part list reference.
+Licensed under [CC-SHAREALIKE](https://creativecommons.org/licenses/by-sa/3.0/)
```

Comment:
You were saying that your board doesn't count as an adaptation because you just referenced a parts list. So you can license under CERN-OHL-P v2 instead of CC-share alike
```suggestion
```

Reply:
My bad, I'll clear up the verbiage- I meant that dtornqvist's project was licensed under CC-ShareAlike :)  I'm happy to just make this breakout part of the overarching CERN-OHL-P v2.